### ISSUE TITLE
Include Nextstrain tutorial videos in docs

### ISSUE THREAD

**Issue by eharkins** [nextstrain/docs.nextstrain.org#78]
Yesterday during a meeting, we talked about how there are a number of existing (and will be more in the future) tutorial-style videos on how to use various parts of nextstrain. 

For example, the following two part series on ncov:
1. [running a build for sars-cov-2](https://www.youtube.com/watch?v=ciGqAYT4d-4&ab_channel=Nextstrain)
2. [visualizing it in auspice](https://www.youtube.com/watch?v=oWHY1plmYIM)

The above videos are great example of the question of how to include such resources in our docs. They are largely directly based on our documentation (in this case, [the ncov tutorial](https://docs.nextstrain.org/projects/ncov/en/latest/index.html)). 

I could imagine it would still be useful to put these videos into docs whether embedded [1] into RTD or as external links, since some folks may prefer video format.

However, it would be good to review these videos to make sure that they are up to date with the current version of the docs they are based on, and potentially update those docs as necessary to include key bits currently only in the videos.

And there's also the question of whether to include them in the tutorials section or have a separate section for workshop-style resources. I would think for simplicity to include them in the tutorials section in a way that clearly highlights them as a supplementary resource or alternative format for the tutorial. 

[1] Embedding youtube videos in read the docs works like this in a `.rst` file (you can also wrap the `iframe` in a `div` to add styling e.g. `margin`):
```
.. raw:: html

    <iframe width="560" height="315" src="https://www.youtube.com/embed/ciGqAYT4d-4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
The html code was copied from youtube by going to https://www.youtube.com/watch?v=ciGqAYT4d-4&ab_channel=Nextstrain and clicking "share" then "embed".

**Comment by cassiawag** [comment]
I agree that including them in the tutorials section as an alternative resource is a good way to go!
